src/blocks/block-coauthors/components/memoized-coauthor-template-block-preview.js

```diff
@@ -0,0 +1,41 @@
+import { memo } from '@wordpress/element';
+import { __experimentalUseBlockPreview as useBlockPreview } from '@wordpress/block-editor';
+
+/**
+ * CoAuthor Template Block Preview
```

Comment:
When is this preview meant to be kicking in? Is this the preview of the "live data" after the block is inserted? Or the block preview from the onhover in the Block Inserter?

Reply:
This is for after the block is inserted.

Its taken from the Post Template Block in WordPress Core. My understanding is that if you're trying to make a template based on a set of blocks and apply it to an array of items ( in their case posts, in our case authors ) then you can only work within a single active item. The others are rendered as inactive previews which receive updates as you modify the active item.